spec/instant.html

```diff
@@ -663,7 +663,7 @@ <h1>
         1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(*"second"*, _smallestUnit_).
         1. Let _largestUnit_ be ? GetTemporalUnit(_options_, *"largestUnit"*, ~time~, *"auto"*).
         1. If _largestUnit_ is *"auto"*, set _largestUnit_ to _defaultLargestUnit_.
-        1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
+        1. If LargerOfTwoTemporalUnits(_largestUnit_, _smallestUnit_) is not _largestUnit_, throw a *RangeError* exception.
```

Comment:
just curious, what happens if theyre the same unit?

Reply:
Then that unit is returned and this step will not throw, which matches the current behavior of [ValidateTemporalUnitRange](https://tc39.es/proposal-temporal/#sec-temporal-validatetemporalunitrange) (i.e., single-unit ranges are allowed).